openmetadata-ui/src/main/resources/ui/src/jsons/en.ts

```diff
@@ -37,6 +37,8 @@ const jsonData = {
 
     'triggering-ingestion-error': 'Error while triggering ingestion workflow',
 
+    'fetch-chart-error': 'Error while fetching charts',
```

Comment:
Nit: To be consistent
```suggestion
    'fetch-chart-error': 'Error while fetching charts!',
```